worlds/Cache.py

```diff
@@ -2,14 +2,15 @@
 from . import BaseWorld
 
 class Cache(BaseWorld):
-    def __init__(self, max_size=None, delay=0):
+    def __init__(self, max_size=None, delay=0, pre_add=lambda trajs, hist:hist):
```

Comment:
Bleah. This breaks data integrity. What about passing numeric distribution parameters?